composeApp/src/commonMain/kotlin/composables/AutoCheckboxes.kt

```diff
@@ -0,0 +1,183 @@
+package composables
```

Comment:
I would like you to split this into four methods, using the two methods in main as a base